faq/index.md

```diff
@@ -99,3 +99,13 @@ For a deeper look at the issues surrounding password theft [read this post by Al
 Password managers are a way to manage and remember large numbers of passwords but they fundamentally don’t solve the security issues with password theft.  A user can still be tricked into handing over their password during a phishing attack. The password manager's ‘master password’ grants access to all username and password combinations stored in the password manager and is vulnerable to the same risks as any other password.  Password managers are a high-value target for hackers.
 
 Using a password manager is a choice the user has to make and, in some cases, pay for.  The website owner has no control over how well the user looks after their passwords either in a password manager or not.  DID.app is different because the website owner chooses to use it to improve user experience and remove the password problem.
+
+### How does DID.app handle privacy?
+
+Authenticating users requires the use of an email address.  Email addresses are considered personally identifiable information.  When an end-user signs in with DID.app, the service collects and stores the user's email address.
```

Comment:
```suggestion
Authenticating users requires them to submit an email address. 
Email addresses are considered personally identifiable information. 
DID.app stores user email address, to provide it's secure authentication service.
```

Putting each sentence on a new line. Also I would avoid "The service" because it could be DID or the third party